src/Interface.jl

```diff
@@ -111,4 +114,3 @@ ymax(grd) = ymin(grd) + ycellsize(grd) * (ycells(grd) - 1)
 
 indices(grd::AbstractGrid) = error("function not defined for $(typeof(grd))")
 indices(grd::AbstractGrid, idx) = error("function not defined for $(typeof(grd))")
-coordinates(grd::AbstractGrid) = error("function not defined for $(typeof(grd))")
```

Comment:
I don't really understand this change - AbstractGrid does not inherit from AbstractPlaces anymore (though the docstring needs to be fixed).

Reply:
I'd forgotten about that change sorry! Will fix...